graph/update_places.go

```diff
@@ -50,9 +57,12 @@ func updatePlacesDetails(searcher *iowrappers.PoiSearcher, places []POI.Place) {
 
 	for placeId, idx := range placeIdToIdx {
 		updatePlaceDetails(&places[idx], m.results[placeId])
+		placesNeedUpdate = append(placesNeedUpdate, places[idx])
 	}
+	return placesNeedUpdate
 }
 
-func updatePlaceDetails(place *POI.Place, details maps.PlaceDetailsResult) {
-	place.URL = details.URL
+func updatePlaceDetails(place *POI.Place, details POI.Place) {
+	place.SetURL(details.GetURL())
+	logrus.Debugf("place with ID: %s URL updated to %s", place.GetID(), place.GetURL())
```

Comment:
Add a special label in the log for easy search [TEST REDIS URL UPDATE]

Reply:
Done